velox/dwio/common/SeekableInputStream.h

```diff
@@ -131,6 +134,7 @@ class SeekableFileInputStream : public SeekableInputStream {
   virtual bool Next(const void** data, int32_t* size) override;
   virtual void BackUp(int32_t count) override;
   virtual bool Skip(int32_t count) override;
+  virtual bool skip(int64_t count) override;
```

Comment:
Have two functions with the same name(`skip` and `Skip`) if we ignore the lowercase/uppercase of the first letter seems odd?

Reply:
My reasoning was:
- Methods we inherit from Google's `ZeroCopyInputStream` are capitalize, while ones we inherit from `SeekableInputStream` aren't.
- The only other name that would make sense would be `skipInt64` or something, which I think is less idiomatic than having an "overloaded" method.

WDYT?

Reply:
`skipInt64` seems more clear for me :). The thing here is `skip` and `Skip` is not an `overload` relationship, people see would come to a question: why two similar functions just diff at: one with a uppercase `S`, one with a lowercase `s`

Reply:
Hey sorry, I forgot about this since I was iterating this PR on Phabricator. Here's a new PR that addresses this: #7240